main/main.script

```diff
@@ -7,6 +7,10 @@ local function print_props(url)
 	print("    .u64", go.get(url, "u64"))
 	print("    .i64", go.get(url, "i64"))
 	print("    .v3", go.get(url, "v3"))
+
+	for i = 1,10 do
+		print("    .array_f32", i, go.get(url, "array_f32", {index=i}))
```

Comment:
Using out relatively new array index feature for `go.get()`

Reply:
Yes, it's dynamic.
Only reason I did it this way was for convenience.
You'll have to handle the out of bounds yourself in Lua.

Reply:
No, we currently have support for that.
We added this indexed support a short while ago.